sw-emulator/lib/periph/src/sha512_acc.rs

```diff
@@ -34,7 +36,7 @@ const SHA_ACC_OP_TICKS: u64 = 1000;
 
 const SHA512_BLOCK_SIZE: usize = 128;
 const SHA512_HASH_SIZE: usize = 64;
-const SHA384_HASH_SIZE: usize = 48;
+const _SHA384_HASH_SIZE: usize = 48;
```

Comment:
I don't think this needs to be _SHA384_HASH_SIZE.

Reply:
It's only used in tests (because Sha512AcceleratorRegs::_hash_len/_copy_hash are only used in tests). So rustc complains if it doesn't have the underscore.

Reply:
Consider using #[cfg(test)]